classes/mender-sdimg.bbclass

```diff
@@ -167,15 +167,18 @@ EOF
     # Call WIC
     IMAGE_CMD_wic
 
+    wicimgname="${IMGDEPLOYDIR}/${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.wic"
```

Comment:
Oh, right I see the poky patch mentions it.

But I expect this will be incompatible with krogoth then? Which means we'll have to diverge the branches, unfortunately.